CMakeLists.txt

```diff
@@ -262,23 +262,23 @@ target_link_libraries(ufs-weather-model-lib PUBLIC fv3atm)
 ###############################################################################
 ### UFS executable
 ###############################################################################
-add_executable(NEMS.exe
+add_executable(ufs_model
```

Comment:
Is the executable name? Can we use ufs_model.exe to avoid confusion?

Reply:
In cmake context this functions declares a target, which will create an executable, and targets typically do not have extension. Another reason is that I think we are moving away from using the .exe suffix (f.e. ufs_utils executables)

Reply:
@aerorahul Do you have an opinion or suggestion about this, since I know you will be updating all install targets in s2s version of this file.

Reply:
We may need more discussion, let's take it as is for now.

Reply:
executables don't have an extension. 
The executable name should reflect the project name.
ufs_model is fine for now for the executable.

Reply:
@aerorahul, "executables don't have an extension." seems just cmake convention, if we put it in a context of test system, giving project name as executable name is confusing as it's hard to know what the file is, real project name or an executable. But this is just my opinion. I see the executable is copied to ${BUILD_NAME}.exe, so I am find  for it in rt.sh

Reply:
what do you mean cmake convention? CMake is not enforcing anything here on the name.  You could name it whatever.
Do you see a `ls.exe` or `grep.exe` or `nc-config.x` or whatever you see in `/usr/bin` or any of the compiled executables such as `ncdump`, `h5dump` or `icc.exe`?

The choice of `.exe` comes from Windows.  That is a convention.
The convention on Linux and Unix is, executables do not have an extension.  They are determined by the permissions on the file.